modules/ROOT/pages/connecting-salesforce-orgs.adoc

```diff
@@ -1,34 +1,54 @@
 = Connect a Trusted Salesforce Organization to Anypoint Platform
 
-Establish a tenant relationship between your Anypoint Platform organization and a trusted Salesforce organization to enable secure data sharing. Initiate the connection in Anypoint Platform and complete it in Salesforce using organization keys. After linking, you can manage capabilities and verify the connected organization details.
+Establish a tenant relationship between your Anypoint Platform organization and a trusted Salesforce organization to enable secure data sharing and centralized capability management. The connection uses two tenant keys (one from each organization) and involves the Anypoint Platform organization administrator and the Salesforce organization administrator. Depending on whether one person holds both roles or two people split the roles, follow the appropriate scenario.
 
 Ways to establish a tenant relationship include:
 
-* A Salesforce organization is provisioned with the tenant relationship with an Anypoint Platform organization already established. In this case, the Anypoint Platform organization administrator just needs to xref:managing-connected-salesforce-orgs.adoc#enable-disable-connection[enable the connection] in Anypoint Platform. 
-* The Anypoint Platform organization administrator initiates the tenant relationship in Anypoint Platform and the Salesforce organization administrator completes the setup in the Salesforce organization.   
-
-In some cases, the Anypoint Platform organization administrator and Salesforce organization administrator are different people, and in some cases, they are the same person.
+* A Salesforce organization is provisioned with the tenant relationship to an Anypoint Platform organization already established. In this case, the Anypoint Platform organization administrator only needs to xref:managing-connected-salesforce-orgs.adoc#enable-disable-connection[enable the connection] in Anypoint Platform.
```

Comment:
Does vale squawk at adding "already" here? Usually, it's unnecessary, but it might help in this case.

Reply:
Or maybe "enable the existing connection"?